buildkite/src/Jobs/Test/TestnetIntegrationTests.dhall

```diff
@@ -29,6 +29,7 @@ in Pipeline.build Pipeline.Config::{
     TestExecutive.execute "payment" dependsOn,
     TestExecutive.execute "delegation" dependsOn,
     TestExecutive.execute "gossip-consis" dependsOn,
+    TestExecutive.execute "opt-block-prod" dependsOn,
```

Comment:
I think this name is too long by 1 character (which is why some of the other names are truncated).

Reply:
Yeah, I know the issue, but as evidenced by green CI, I'm not surpassing the limit